### PR TITLE
ENH: Add Python solid clip example

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add Python solid clip example.

### CODE REVIEW

File: src/Python/Meshes/SolidClip.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    # Create a property to be used for the back faces. Turn off all
+    # shading by specifying 0 weights for specular and diffuse. Max the
+    # ambient.
+    back_faces = vtk.vtkProperty()
+    back_faces.SetSpecular(0.0)
+    back_faces.SetDiffuse(0.0)
+    back_faces.SetAmbient(1.0)
+    back_faces.SetAmbientColor(1.0000, 0.3882, 0.2784)
+
```
Comment (ajpmaclean) [inline src/Python/Meshes/SolidClip.py:38]: In the "old days" this was acceptable. Can you please use vtkNamedColors instead.

File: src/Python/Meshes/SolidClip.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    render_window.AddRenderer(renderer)
+
+    render_window_interactor = vtk.vtkRenderWindowInteractor()
+    render_window_interactor.SetRenderWindow(render_window)
+
+    # Add the actor to the renderer
+    renderer.AddActor(superquadric_actor)
+    render_window.Render()
+
```
Comment (ajpmaclean) [inline src/Python/Meshes/SolidClip.py:55]: Add a window title here.

``` python
    render_window.Render()
    render_window.SetWindowName('SolidClip')
```

**Comment by ajpmaclean** [comment]
An image is needed for `Testing/Baseline/Python/Meshes`.

**Comment by ajpmaclean** [comment]
One further comment! Could you pleas add a background to the image, once again using `vtkNamedColors`. I'll backport your changes to the C++ example. So it will also look nicer!

File: src/Python/Meshes/SolidClip.py (hunk 2 above)
Comment (jhlegarreta) [reply src/Python/Meshes/SolidClip.py:55]: Do you mean to move [this line](https://github.com/lorensen/VTKExamples/pull/659/files/28c012ed57a68e8098a5995812eebc528025902b#diff-4abd4f98cb238ac5165ee63bb467a3aaR45) to where you pointed?

File: src/Python/Meshes/SolidClip.py (hunk 1 above)
Comment (jhlegarreta) [reply src/Python/Meshes/SolidClip.py:38]: Shall I directly use this:
```
from vtk.util.colors import tomato
(...)
back_faces..SetDiffuseColor(tomato)
```
or do you prefer me to do
```
colors = vtk.vtkNamedColors()
back_faces.SetAmbientColor(colors.GetColor3d("tomato").GetData())
```
?

Comment (ajpmaclean) [reply src/Python/Meshes/SolidClip.py:38]: Olease use vtkNamedColors, in the case of Python, you don't need the ` ,GetData()`

File: src/Python/Meshes/SolidClip.py (hunk 2 above)
Comment (ajpmaclean) [reply src/Python/Meshes/SolidClip.py:55]: SetWindowName goes anywhere after the first render.

Comment (jhlegarreta) [reply src/Python/Meshes/SolidClip.py:58]: Done in b6ca8de.

File: src/Python/Meshes/SolidClip.py (hunk 1 above)
Comment (jhlegarreta) [reply src/Python/Meshes/SolidClip.py:40]: Done in b6ca8de.

**Comment by jhlegarreta** [comment]
> An image is needed for Testing/Baseline/Python/Meshes.

Added in b6ca8de. I had not realized (or forgot) that examples get testes. It is awesome !

> One further comment! Could you please add a background to the image, once again using vtkNamedColors

Done in b6ca8de.

>  I'll backport your changes to the C++ example. So it will also look nicer!

Thanks Andrew.

**Comment by ajpmaclean** [comment]
Thanks for the work. I'll add it to the menu and do a couple of little changes and backport these changes to the C++ code. It looks good.

**Comment by jhlegarreta** [comment]
Sounds good. Thanks Andrew.

**Comment by ajpmaclean** [comment]
I have just finished, here is the new image: 
TestSolidClip
I hope you like the ghost of what has been clipped away. The C++ code and indexes have also been updated. Thanks for all your efforts.

**Comment by jhlegarreta** [comment]
Awesome. I love it !! Thanks for having done that @ajpmaclean !

**Pull request merged**